caffe2/utils/math.h

```diff
@@ -610,19 +610,19 @@ void CopyMatrix(
     Context* context);
 
 template <typename T, class Context>
-void CopyVector(const int N, const T* A, T* B, Context* context);
+void CopyMatrix(
+    const int M,
+    const int N,
+    const T* A,
+    const int A_outer_stride,
```

Comment:
Add comments on A/B_outer/inner_stride ?

Reply:
I think the actual impact of the memory part will be less than we estimated. Since currently in the inference part we are using shared col_buffer, so the peek usage of that part will depends on the largest layer. If the whole model has at least one layer with G = 1, the actually usage of memory will not increase a lot.

Reply:
Sorry that I was not clear. I was mainly talking about cache locality rather than the maximum col_buffer size.